### DISCUSSION TITLE
Welcome to atrilabs-engine Discussions!

### DISCUSSION THREAD

**Discussion by darshitac11** [Atri-Labs/atrilabs-engine#158] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jonathanalvares9009** [comment]
Hi I'm interested in the project could I be added as a contributor?

**Comment by darshitac11** [comment]
You are automatically listed as a contributor if your Pull Request is merged. 

If you want to be an open source member of the core development team, then you'll be expected to contribute more frequently by resolving open issues. 

I hope this answers your question. Let me know if you need more information.

**Reply by jonathanalvares9009** [reply]
Thanks
